pkg/api/controller.go

```diff
@@ -41,8 +42,28 @@ type Controller struct {
 func NewController(config *config.Config) *Controller {
 	var controller Controller
 
-	logger := log.NewLogger(config.Log.Level, config.Log.Output)
+	if config != nil && config.Extensions != nil {
+		defaultTrue := true
```

Comment:
"defaultTrue" usage here is weird? linter forcing you? what is the complaint?

Reply:
will use 
`var defaultVal = true` 
instead